contracts/libraries/CoreLib.sol

```diff
@@ -22,50 +22,44 @@
 pragma solidity ^0.6.2;
 
 /**
- * @title   Library for Core Logic for Primitive Option tokens.
+ * @title   Library for calculating different proportions of long and short Option tokens.
```

Comment:
Are we capitalizing the term 'Option'?  Might need to use consistent naming conventions, I just invented 5 different terms today and its gonna get messy if we capitalize non-proper nouns.